src/components/pill/pill-sizes.style.js

```diff
@@ -0,0 +1,47 @@
+export default (css, isClassic, isDeletable) => ({
```

Comment:
I'm really struggling to work out what this file is / does? Is it meant to be in here? As far as I can tell, it isn't being imported anywhere, and wouldn't work because it has problems (uses methods like `css`, `isClassic` when they haven't been imported etc)? Feels like I'm missing something